configure.ac

```diff
@@ -111,6 +111,15 @@ AC_ARG_ENABLE([driver-xgvr],
 
 AM_CONDITIONAL([BUILD_DRIVER_XGVR], [test "x$driver_xgvr_enabled" != "xno"])
 
+# VR-Tek HMD driver
+AC_ARG_ENABLE([driver-vrtek],
+        [AS_HELP_STRING([--disable-driver-vrtek],
+                [disable building of vr-tek driver [default=yes]])],
+        [driver_vrtek_enabled=$enableval],
+        [driver_vrtek_enabled='yes'])
+
+AM_CONDITIONAL([BUILD_DRIVER_VRTEK], [test "x$driver_vrtek_enabled" != "xno"])
```

Comment:
We plan to remove autotools support. I think this should be done before this series is merged, so the configure.ac and Makefile.am changes can be removed.

Reply:
Agree, since this will be pulled after 0.3 release, autotools is not required anymore.

Reply:
OK got it. I just pushed a commit that removes this. Thanks!